### DISCUSSION TITLE
How to get report of bounces from postal to email sending application?

### DISCUSSION THREAD

**Discussion by kayastha1988** [postalserver/postal#2361] [Feature suggestions]
While sending email, I can see multiple email address as bounce or invalid. How to show them into my email sending environment without looking into my postal mail server?

![image](https://user-images.githubusercontent.com/19544006/221884445-e8eedf03-df1d-4c46-98dd-ce9f3c3b8adf.png)

**Comment by willpower232** [comment]
You can implement webhooks https://docs.postalserver.io/developer/webhooks

**Reply by kayastha1988** [reply]
Hell Will, thanks for your response. I have checked this webhooks part but still I am confused and unable to implement it. I have different environment from where I used to send emails for email broadcasting. 

MAILSERVER.MYDOMAIN.COM is where I have setup Postal SMTP and 
EMAIL.ANOTHERDOMAIN.COM is my email sending application

I wonder how to catch those bounce email records from my postal smtp server to my email application where I can see the number of bounce emails.

For reference I have attached image below. The email which was bounce in postal, trying show those email in below email application where bounces show 0. 

Looking forward to hearing from you.

![image](https://user-images.githubusercontent.com/19544006/221907724-d36c8dac-2cdc-440f-b03f-ed6436a96b92.png)

**Reply by willpower232** [reply]
You will need to make your email sending application capable of understanding Postal webhooks. Postal will then be able to send your email sending application a message when the status of an email changes and your application will know what the status of the message is.

**Reply by kayastha1988** [reply]
I have created webhooks as below image. 

Just wanted to know if there is any webhooks or MTA config file where I could declare my email application bounce processing so that I email application could tell to postal or vice versa regarding bounce records

![image](https://user-images.githubusercontent.com/19544006/221913052-324d8b55-49f7-425a-b6a1-ceb82e68181b.png)
